Makefile.kokkos

```diff
@@ -1,5 +1,9 @@
 # Default settings common options.
 
+ifneq ($(KOKKOS_USE_DEPRECATED_MAKEFILES), 1)
```

Comment:
What about `KOKKOS_I_HEREBY_AKNOWLEDGE_THAT_I_UNDERSTAND_THAT_SUPPORT_FOR_MAKEFILES_WAS_DROPPED_AND_WILL_BE_REMOVED_SOON`

Reply:
yeah, why not using Shakespeare's Hamlet in all caps and underscores